tests/TestCase/BEditaClientTest.php

```diff
@@ -256,4 +256,60 @@ public function testSaveObject()
         static::assertArrayHasKey('attributes', $response['data']);
         static::assertEquals($data['title'], $response['data']['attributes']['title']);
     }
+
+    /**
+     * Test `upload` and `createStreamMedia` methods
+     *
+     * @return void
+     *
+     * @covers ::get()
+     * @covers ::upload()
```

Comment:
Here we should cover only `::upload()` and `::createStreamMedia()` other methods are tested in its own test methods